modules/images/image-loading-optimization/detection.php

```diff
@@ -35,6 +35,9 @@ function ilo_get_detection_script( string $slug, array $needed_minimum_viewport_
 	 */
 	$detection_time_window = apply_filters( 'ilo_detection_time_window', 5000 );
 
+	$web_vitals_version = require_once __DIR__ . '/detection/web-vitals/index.asset.php';
```

Comment:
@westonruter Do we need to add some notice in case this file does not exist? given it will throw a fatal error if doesn't exist and the user will be required to run `npm run build`.

Reply:
Yes, the plugin should short-circuit with an error if the file doesn't exist. Similar to AMP plugin or PWA plugin.

Reply:
https://github.com/WordPress/performance/pull/953/commits/70851eb33681b858249b0a0f07a4d5181daee7c2